Gemfile

```diff
@@ -9,6 +9,8 @@ gem 'execjs'
 gem 'therubyracer', platforms: :ruby
 gem 'uglifier', '~> 2.7'
 
+gem 'mumuki-domain', github: 'mumuki/mumuki-domain', branch: 'feature-add-uppercase-mode'
```

Comment:
This PR is very nice. However, since it will be the first of its kind - let's call it _user_preferences_ - I would like to properly define its architecture.

Reply:
Sure, let's do that.